lib/Parse/ParseDecl.cpp

```diff
@@ -6236,7 +6274,8 @@ void Parser::ParseParameterDeclarationClause(
 
       // Inform the actions module about the parameter declarator, so it gets
       // added to the current scope.
-      ParmVarDecl *Param = Actions.ActOnParamDeclarator(getCurScope(), ParmDeclarator);
+      ParmVarDecl *Param =
```

Comment:
This looks like only a formatting change not related to parsing checked scopes.  Please undo this change because it might make merging updated sources for clang harder.